system/jhtree/jhtree.hpp

```diff
@@ -256,10 +251,6 @@ interface IIndexLookup : extends IInterface // similar to a small subset of IKey
     virtual const void *nextKey() = 0;
     virtual unsigned __int64 getCount() = 0;
     virtual unsigned __int64 checkCount(unsigned __int64 limit) = 0;
-    virtual unsigned querySeeks() const = 0;
```

Comment:
is querySeeks() (and other methods) in jhtree.ipp still needed/used?

Reply:
These will be removed in separate jira: https://track.hpccsystems.com/browse/HPCC-29817